sources/packages/web/src/services/StudentAppealService.ts

```diff
@@ -35,20 +37,43 @@ export class StudentAppealService {
     });
   }
 
+  /**
+   * Get student application appeal.
+   * @param appealId appeal id.
+   * @returns student application appeal.
+   */
+  async getStudentAppealWithRequests(
+    appealId: number,
+  ): Promise<StudentAppealAPIOutDTO<StudentAppealRequestAPIOutDTO>>;
+
```

Comment:
I would prefer to have the blank line removed to give a better idea that it is a single method.